laundry/api_wrapper.py

```diff
@@ -0,0 +1,200 @@
+import csv
+import os
+
+import pkg_resources
+import requests
+from bs4 import BeautifulSoup
+
+
+LAUNDRY_URL = os.environ.get("LAUNDRY_URL", "http://suds.kite.upenn.edu")
+ALL_URL = "{}/?location=".format(LAUNDRY_URL)
```

Comment:
I know this was directly copied from LAS but it can be optimized to `= f"{LAUNDRY_URL}/?location="`